website/docs/r/dynatrace_monitors.html.markdown

```diff
@@ -0,0 +1,125 @@
+---
+subcategory: "Dynatrace"
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_dynatrace_monitor"
+description: |-
+  Manages Dynatrace monitors.
+---
+
+# azurerm_dynatrace_monitors
+
+Manages Dynatrace monitors.
+
+## Example Usage
+
+```hcl
+
+resource "azurerm_resource_group" "example" {
+  name     = "example-resources"
+  location = "West Europe"
+}
+
+resource "azurerm_dynatrace_monitor" "example" {
+  name                            = "exmpledynatracemonitor"
+  resource_group_name             = azurerm_resource_group.example.name
+  location                        = azurerm_resource_group.test.location
+  monitoring_status               = "Enabled"
+  marketplace_subscription_status = "Active"
+
+  identity {
+    type = "SystemAssigned"
+  }
+
+  user {
+    first_name   = "Alice"
+    last_name    = "Bobab"
+    email        = "alice@microsoft.com"
+    phone_number = "123456"
+    country      = "westus"
+  }
+
+  plan {
+    usage_type     = "COMMITTED"
+    billing_cycle  = "MONTHLY"
+    plan           = "azureportalintegration_privatepreview@TIDhjdtn7tfnxcy"
+    effective_date = "2019-08-30T15:14:33Z"
+  }
+}
+```
+
+## Arguments Reference
+
+The following arguments are supported:
+
+* `name` - (Required) Name of the Dynatrace monitor. Changing this forces a new resource to be created.
+
+* `resource_group_name` - (Required) The name of the Resource Group where the Dynatrace monitor should exist. Changing this forces a new resource to be created.
+
+* `location` - (Required) The Azure Region where the Dynatrace monitor should exist. Changing this forces a new resource to be created.
+
+* `identity` - (Required) The kind of managed identity assigned to this resource.  A `identity` block as defined below.
+
+* `marketplace_subscription` - (Required) Flag specifying the Marketplace Subscription Status of the resource. If payment is not made in time, the resource will go in Suspended state. Possible values are `Active`, `Suspended`.
+
+* `plan` - (Required) Billing plan information. A `plan` block as defined below. Changing this forces a new resource to be created.
+
+* `user` - (Required) User's information. A `user` block as defined below. Chainging this forces a new resource to be created.
+
+* `monitoring_status` - (Optional) Flag specifying if the resource monitoring is enabled or disabled. Possible values are `Enabled`, `Disabled`.
+
+* `tags` - (Optional) A mapping of tags to assign to the resource.
+
+---
+
+A `identity` block supports the following:
+
+* `type` - (Required) The type of identity used for the resource. Only possible value is `SystemAssigned`.
+
+---
+
+A `plan` block supports the following:
+
+* `billing_cycle` - (Optional) Different billing cycles. Possible values are `MONTHLY`, `WEEKLY`.
+
+* `effective_date` - (Required) Date when plan was applied.
+
+* `plan` - (Required) Plan id as published by Dynatrace.
```

Comment:
Is there a source of the possible values of the `plan`? The one used in the acctest seems not something users can figure out?